src/client.cpp

```diff
@@ -939,7 +939,7 @@ void writePlayerPos(LocalPlayer *myplayer, ClientMap *clientMap, NetworkPacket *
 	u32 keyPressed   = myplayer->keyPressed;
 	// scaled by 80, so that pi can fit into a u8
 	u8 fov           = clientMap->getCameraFov() * 80;
-	u8 wanted_range  = clientMap->getControl().wanted_range / MAP_BLOCKSIZE;
+	u8 wanted_range  = (clientMap->getControl().wanted_range - 1) / MAP_BLOCKSIZE + 1;
```

Comment:
I just noticed that wanted_range is in fact already a float. I assumed it was integral...

Changed to ceil().